### ISSUE TITLE
Op info test for `logspace .. masked.amax`

### ISSUE THREAD

**Issue by qihqi** [pytorch/xla#7507]
## Fix the Op info test for `logspace .. masked.amax`

1. Find the lines 153 to 157 of [test_ops.py](test/test_ops.py) and remove
   `logspace` .. `masked.amax` from `skip_list`
2. Run op_info test with `pytest test/test_ops.py`
3. Fix the failure.

Please refer to 
[this guide](https://github.com/pytorch/xla/blob/master/experimental/torch_xla2/docs/fixing_op_info_test.md) as guide to fix:

Also refer to these PRs:
* https://github.com/pytorch/xla/pull/7343
* https://github.com/pytorch/xla/pull/7312
* https://github.com/pytorch/xla/pull/7325

**Comment by ManfeiBai** [comment]
actual work will be "logspace", "lu", "lu_solve", "lu_unpack"

**Comment by ManfeiBai** [comment]
Hi, @barney-s, thanks for contribution! This issue would be a new first issue, please feel free to reply this issue, so that we could assign it to you

**Comment by barney-s** [comment]
Please assign it to me.

**Comment by barney-s** [comment]
### Looking at `logspace`

`logspace` is decomposing to `linspace`.  We will fix `linspace` https://github.com/pytorch/xla/issues/7505 and revisit this again.

Script:
```python
import torch
import torch_xla2

env = torch_xla2.default_env()
env.config.debug_print_each_op = True
env.config.debug_accuracy_for_each_op = True

with env:
  y = torch.tensor([1, 5, 10])
  print(torch.logspace(start=-10, end=10, steps=5))
```

Output:

```
FUNCTION: tensor
/usr/local/google/home/barni/workspace/pytorch/xla/experimental/torch_xla2/torch_xla2/ops/jtorch.py:48: UserWarning: Explicitly requested dtype <class 'jax.numpy.int64'> requested in array is not available, and will be truncated to dtype int32. To enable more dtypes, set the jax_enable_x64 configuration option or the JAX_ENABLE_X64 shell environment variable. See https://github.com/google/jax#current-gotchas for more.
  return jnp.array(
FUNCTION: logspace
 DISPATCH: aten::logspace
  FUNCTION: linspace
   DISPATCH: aten::linspace
    FUNCTION: arange
     DISPATCH: aten::arange.start_step
    FUNCTION: lt
     DISPATCH: aten::lt.Scalar

```

**Comment by barney-s** [comment]
https://github.com/pytorch/xla/pull/8236

**Comment by ManfeiBai** [comment]
reopen due to we still have `lu_unpack`

**Comment by barney-s** [comment]
https://github.com/pytorch/xla/pull/8262 for `lu_unpack`
